angular_forms/CHANGELOG.md

```diff
@@ -1,7 +1,13 @@
+### New Features
+
+*   `Control.markAsPristine` added. This will clear the `dirty` property.
+
 ### Breaking Changes
 
 *   `NgControlName` will no longer initialize with `null` if a value is
     specified by 'ngModel'.
+    
+*   The `touched` property of `Control`s is now propagated to parents / children.
```

Comment:
https://github.com/dart-lang/angular/pull/1294/files#diff-6292b1e8999dc3001869afd57e3648c4R87 says that it is propagated up to ancestors, not down to children. Which is it?